File: class/9.24/triangle.c
#include <stdio.h>
#include <math.h>
int main()
{
    double s,area;
    int a,b,c;
    scanf("%d%d%d",&a,&b,&c);
    s=(a+b+c)/2.0;
    area=sqrt(s*(s-a)*(s-b)*(s-c));
    printf("a=%d\tb=%d\tc=%d\n",a,b,c);
    printf("area is %.2lf\n",area);
    return 0;
}

File: java/Triangle.java
class Triangle {
    double a, b, c;
    double C, area;

    Triangle(double a, double b, double c) {
        this.a = a;
        this.b = b;
        this.c = c;
    }
    double area() {
        double s = (a + b + c) / 2;
        area = Math.sqrt(s * (s - a) * (s - b) * (s - c));
        return area;
    }
    double C() {
        C = a + b + c;
        return C;
    }
    public static void main(String[] args) {
        Triangle t = new Triangle(3, 4, 5);
        System.out.println("Area: " + t.area());
        System.out.println("Circumference: " + t.C());
    }
}